### ISSUE TITLE
Drop rate config may not work as expected

### ISSUE THREAD

**Issue by PitchBright** [GrowthcraftCE/Growthcraft-1.7#56]
Rice/Grape/Hops 

I find the drop rate in the config files a bit confusing. Using Rice as an example:

```
# [Higher -> Rarer] Default : 3
I:"Rice grass drop rarity"=3
```

I find that Rice drops too often from grass, so I tried setting it to 100. It seemed to drop as often, if not, more often than the default setting of 3. Then I tried setting it to 0, and now Rice doesn't drop at all.

The way the description is worded, I'm led to believe that the higher I set the number, the less frequently I should see Rice drop from grass. Can we get a better description, and/or definition on what the values actually mean? (ie. 3 = 1 in 3 chance… or 3 out of every 16 blocks broken… or 3% chance)

**Comment by IceDragon200** [comment]
Wheat is `10` by the way, I've been following where this value goes, I've ended up all the way into [forge](https://github.com/MinecraftForge/MinecraftForge/blob/1.7.10/src/main/java/net/minecraftforge/common/MinecraftForge.java#L42) still tracing it.

After looking through it, I've ended up in the minecraft codebase with `WeightedRandom`, and I still have no idea how it works.
#Java

The most I can tell you is that `rarity` is actually `weight`, I'm guessing it behaves like a ratio (1/10 chance of dropping), however that rate varies depending on the number of other grass seeds present, I guess, so that `1/10`, might actually be `1/20`, or however this math stuff works.

So I don't believe this is a bug, more of bad luck (or good luck? It's so rare for me, I have more diamonds than rice!), also the result may be the same if you keep testing it in the same world, or same seed.

**Comment by PitchBright** [comment]
Aight cool.

It was actually just the description in the config that I was reporting as an issue. I feel like it should be worded in a way that makes (more) sense, or is more descriptive.

**Comment by Alatyami** [comment]
So, looking at the the dev tutorials the description is actually backwards. I've looked at some of the dev tutorials and they complete support the explaination of @IceDragon200 

As a fix, we'll correct the description for the config.

**Comment by Alatyami** [comment]
Resolved via #77

**Comment by Ar97x** [comment]
One guy reported this on the forum. Will it be on the next release. And
when will it be released?
